package-manifest.toml

```diff
@@ -16,5 +17,10 @@ source.rust.binary_names = ["crucible-pantry"]
 source.rust.release = true
 source.paths = [
   { from = "pantry/smf/pantry.xml", to = "/var/svc/manifest/site/crucible/pantry.xml" },
+  { from = "tools/dtrace/upstairs_info.d", to = "/opt/oxide/dtrace/upstairs_info.d" },
```

Comment:
I think `from` can be a directory, if we want to ship the whole thing

Reply:
I'm going to continue to iterate on these, but yeah, just including them all might not be too bad of an idea.